packages/httpsnippet-client-api/test/__datasets__/multipart-data/mock.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  scope: 'https://httpbin.org',
+  method: 'POST',
+  path: '/anything',
+  body: /--form-data-boundary-(.*)\r\nContent-Disposition: form-data; name="foo"; filename="hello.txt"\r\nContent-Type: text\/plain\r\n\r\nHello world!\n\r\n--form-data-boundary-(.*)--\r\n\r\n/,
```

Comment:
`multipart/form-data` boundaries are always unique so I needed to regex this.

The contents of this payload are coming from the contents of `test/__fixtures__/files/hello.txt` as well. File uploads, they work!